sw/device/lib/dif/dif_aes_unittest.cc

```diff
@@ -7,7 +7,6 @@
 #include "gtest/gtest.h"
 #include "sw/device/lib/base/mmio.h"
 #include "sw/device/lib/base/testing/mock_mmio.h"
-#include "sw/device/lib/testing/entropy_testutils.h"
```

Comment:
I should have removed it. Sorry.

Reply:
I needed to learn all the stuff I found figuring it out at some point anyway!